src/sage/matrix/matrix_rational_sparse.pyx

```diff
@@ -819,8 +828,74 @@ cdef class Matrix_rational_sparse(Matrix_sparse):
             [1 0 0]
             [0 1 0]
             [0 0 1]
+
+        .. SEEALSO::
+
+            :meth:`~sage.matrix.matrix_rational_dense.Matrix_rational_dense._right_kernel_matrix`
+        """
+        algorithm = kwds.get('algorithm', 'default')
```

Comment:
use the normal Python way to specify default parameter. Also let the default be `None`.

Reply:
The way I did it is pretty much how it is already done in other parts of the code:

https://github.com/sagemath/sage/blob/cb030433d73cb3a25d5a8c79c5e0c3b1f41893b3/src/sage/matrix/matrix_integer_dense.pyx#L2688-L2690

These methods starting with an underscore are internal and don't show up in the online documentation, so I'd prefer to keep this part as it is, if that's alright.